test/runtests.jl

```diff
@@ -12,8 +12,13 @@ m = Matrix(h)
 mktemp("data") do path, _
     println(path)
     hg_save(path, h)
-    @test read(path, String) == replace(read("data/test1.hgf", String), "\r\n" => "\n")
-end
+    #@test read(path, String) == replace(read("data/test1.hgf", String), "\r\n" => "\n")
+    @test replace(read(path, String), r"\n*$" => "") ==
```

Comment:
It would be simpler to create several test files and try reading them in and check if they produce an equal result. Also can you please test a malformed comment.

As a side note: you can pass several `Pair`s to `replace` in one call.